src/installer.ts

```diff
@@ -84,6 +84,7 @@ export async function installJDK(
         if (value == "JAVA_HOME") core.addPath(path.join(toolPath, "bin"));
         core.exportVariable(value, toolPath);
     });
+    core.exportVariable("TEST_JDK_HOME", toolPath);
```

Comment:
Could you elaborate on why you want to add this? I would prefer not to "pollute" CI environments by doing unexpected stuff, especially without a way to opt-out. (Btw, `TEST_JDK_HOME` can already be set by passing `targets` to the action's configuration.)

If `TEST_JDK_HOME` is commonly used (which I'm not sure it is), it would probably be sufficient to automatically set `TEST_JDK_HOME` whenever `JAVA_HOME` is in `targets`.